python/cugraph/cugraph/dask/community/egonet.py

```diff
@@ -63,10 +63,15 @@ def consolidate_results(df, offsets):
 def convert_to_cudf(cp_arrays):
     cp_src, cp_dst, cp_weight, cp_offsets = cp_arrays
 
+    print(cp_weight)
```

Comment:
debug print?

I see a few others, but I'll just re-review instead of pointing each one out.